ckanext/showcase/logic/action/get.py

```diff
@@ -78,10 +79,14 @@ def showcase_package_list(context, data_dict):
         # for each package id, get the package dict and append to list if
         # active
         for pkg_id in pkg_id_list:
-            pkg = toolkit.get_action('package_show')(context, {'id': pkg_id})
-            if pkg['state'] == 'active':
-                pkg_list.append(pkg)
-
+            try:
+                pkg = toolkit.get_action('package_show')(context,
+                                                         {'id': pkg_id})
+                if pkg['state'] == 'active':
+                    pkg_list.append(pkg)
+            except NotAuthorized:
+                log.error(
```

Comment:
Thanks for this fix, I have just encountered this on a client's website.

I suggest logging as no higher than INFO and probably just DEBUG, as it's not an error, simply a normal situation that private or deleted datasets (or any that require additional privileges) be ignored.